earth_enterprise/src/server/geserver

```diff
@@ -83,8 +83,11 @@ stop() {
   $gehttpd_root/gehttpd_init stop
   echo
   echo $dashes
-  run_as_user $GEPGUSER "$gebin_root/pg_ctl -D /var/opt/google/pgsql/data -l /var/opt/google/pgsql/logs/pg.log stop";
-
+  # only stop if already running...
+  if test -f /var/opt/google/pgsql/data/postmaster.pid ; then
```

Comment:
Normally, we use the convention of `[ -f filename ]` instead of `test`. Is there a reason for using `test` here instead?

Reply:
No reason except personal habit...easy enough to change.